src/components/ProfileTree/ProfileTree.js

```diff
@@ -48,6 +52,8 @@ export const ProfileTree = memo(({
 
   // const ChangesContext = React.createContext(changesCount);
```

Comment:
Remove commented code if it is not really needed.

Reply:
It is not unused code but a part of a bug.
It should remain here until it will be fixed.

Reply:
Please add TODO note then.